### DISCUSSION TITLE
Trivy support for CIS EKS Benchmark

### DISCUSSION THREAD

**Discussion by thevsed** [aquasecurity/trivy#4980] [Ideas]
### Description

Background
At this time Trivy supports the CIS Kubernetes Benchmarks but not the CIS EKS Benchmark. Hopefully this would not be too demanding to add to Trivy because kube-bench already has support for CIS EKS Benchmarks.

Request
I would like to suggest that support for the CIS EKS Benchmark is implemented within Trivy.

### Target

Kubernetes

### Scanner

Misconfiguration

**Comment by chen-keinan** [comment]
@thevsed converted to issue #5638
